tools/serve/serve.py

```diff
@@ -923,6 +923,48 @@ def build_config(override_path=None, config_cls=ConfigBuilder, **kwargs):
     return rv
 
 
+def check_environ(logger, config):
+    is_windows = platform.uname()[0] == "Windows"
+
+    expected_hosts = set(config.domains_set)
+    if is_windows:
+        expected_hosts.update(config.not_domains_set)
+
+    missing_hosts = set(expected_hosts)
+    if is_windows:
+        hosts_path = r"%s\System32\drivers\etc\hosts" % os.environ.get("SystemRoot", r"C:\Windows")
+    else:
+        hosts_path = "/etc/hosts"
+
+    if os.path.abspath(os.curdir) == repo_root:
+        wpt_path = "wpt"
+    else:
+        wpt_path = os.path.join(repo_root, "wpt")
+
+    with open(hosts_path, "r") as f:
+        for line in f:
+            line = line.split("#", 1)[0].strip()
+            parts = line.split()
+            hosts = parts[1:]
+            for host in hosts:
+                missing_hosts.discard(host)
+        missing_hosts = True
```

Comment:
This looks wrong - `missing_hosts` is a set up above, and this will also make :952 always true

Reply:
Oops, leftover debugging code